docs/adr/adr_improved_frontend_dx.md

```diff
@@ -0,0 +1,55 @@
+# ADR: Improved Frontend Developer Experience
+Decision Deadline: As soon as possible
+
+Author: [Felipe](https://github.com/fhenrich33)
+
+## Status
+
+Proposed, Implementing
+
+## Context
+Frontend development experience in its current form is quite lacking. Imposes a lot of artificial difficulties, tooling doesn't work as expected, and adds friction to our FE contributors.
```

Comment:
"quite lacking" and "artificial difficulties" needs to be more specific

Reply:
I have mentioned it briefly here: https://github.com/boxwise/boxtribute/pull/1400/files#r1663184123

But although I can get the app up and running quickly, my local environment doesn't identify things and play well with the code as I expect it to work. TS, ESlint, and Prettier don't work at all without a lot of fiddling. 

I'll accentuate that in the ADR.

Reply:
@jlin95

Reply:
I replied in one of the discussion threads